### ISSUE TITLE
The performance of semantic segmentation on S3DIS dataset

### ISSUE THREAD

**Issue by shuluoshu** [charlesq34/pointnet2#15]
Hi, @charlesq34 , I trained the PointNet++ model by using the dataset S3DIS and the training code from pointnet , with the model "PointNet++", namely, the Pointnet2_sem_seg.py, however, I found no better performance than the original PointNet, so could you offer me the result (mean IoU and Accuracy) for comparison? thanks a lot!

**Comment by charlesq34** [comment]
Hi @shuluoshu 
Have you tried to use XYZ information only?
the RGB and X'Y'Z' features can be quite dominant in S3DIS dataset

**Comment by shuluoshu** [comment]
Hi, @charlesq34 , do you mean that  by comparing the results of PointNet (XYZ instead of original XYZRGB) and PointNet++ only using the XYZ information? Does the PointNet++ model not support RGB information?

**Comment by jefft255** [comment]
Hello,

After getting PointNet++ to work with S3DIS, I experienced the same thing. However, as Charles pointed out, PointNet uses XYZ RGB X'Y'Z' for segmentation, while, as of right now, PointNet++ sementic segmentation model (pointnet2_sem_seg.py) only uses XYZ. The fact that PointNet++ performs in the same ballpark as PointNet using so much less information is quite impressive in my opinion.

 Having read the paper carefully, I do not see a fundamental difficulty in using other features in this model. It would be quite a bit of work to implement however.
